src/bp-templates/bp-nouveau/buddypress-functions.php

```diff
@@ -122,7 +122,7 @@ function() {
 			);
 
 			// When BP Classic is activated, regular themes need this filter. 
-			if ( function_exists( 'bp_classic' ) ) {
+			if ( function_exists( 'bp_classic' ) || is_plugin_active( 'bp-classic/bp-classic.php' ) || is_plugin_active( 'buddyclassic/bp-classic.php' ) ) {
```

Comment:
There are many places where we are doing similar checks that we should update too: https://github.com/search?q=org%3Abuddypress+function_exists(+%27bp_classic%27+)&type=Code

Maybe create a new function to check if the BP Classic plugin is active?! And reuse it where applicable.

Reply:
@renatonascalves Thanks for the detailed feedback, I have implemented the suggestions can you please take a look.

Reply:
@renatonascalves @imath Please let me know if there is any feedback for me.